spec/integration/basic_spec.rb

```diff
@@ -1,24 +1,24 @@
-require 'spec_helper'
+require_relative '../spec_helper'
```

Comment:
There never was the need in ruby for `require_relative`... , just add the folder you want in the load path.
Here `require  spec_helper` should work seamlessly.

Reply:
OK will fix